webpack.config.babel.js

```diff
@@ -282,6 +283,12 @@ const configDist = {
 
   plugins: [
     ...basePlugins('dist'),
+    new CopyPlugin({
```

Comment:
You don't need such, because you're using Sass and did include both core and a day theme already. This one only is needed for anyone who's not using Sass and would want to use pure CSS.

Reply:
Ah, ok. Will remove it.